Exercise_A/index.html

```diff
@@ -7,10 +7,14 @@
     <meta name="description" content="" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="stylesheet" href="" />
+    <title>Home</title>
   </head>
   <body>
     <!-- CODE HERE -->
-
+    <h1>Home</h1>
+    <a href="about.html">
+      <img src="https://twinfinite.net/wp-content/uploads/2021/04/one-punch-man.jpg">
+    </a>
     <script src="" async defer></script>
   </body>
 </html>
```

Comment:
You did a good job adding an anchor tag and linking it to another HTML page. Well done! Also, I love the picture of one punch man, keep pushing yourself until you become the best developer on this planet.